lewis_emulators/keithley_2700/interfaces/stream_interface.py

```diff
@@ -212,3 +216,23 @@ def get_scan_channels(self):
     def set_scan_channels(self, start, end):
         self._device.scan_channel_start = start
         self._device.scan_channel_end = end
+
+    def get_multicmds(self, command, other_commands):
```

Comment:
The .proto file does not seem to be outputting multiple commands separated by semi-colons. If this is needed then it needs associated tests. If it's not used then maybe it doesn't need to exist.

Reply:
This is again for VI compatibility and is covered by testing the emulator with the VI